lib/vrf.c

```diff
@@ -1028,8 +1031,11 @@ int vrf_bind(vrf_id_t vrf_id, int fd, const char *ifname)
 
 	/* can't bind to a VRF that doesn't exist */
 	vrf = vrf_lookup_by_id(vrf_id);
-	if (!vrf_is_enabled(vrf))
+	if (!vrf_is_enabled(vrf)) {
+		zlog_debug("ifname: %s(%u) for vrf: %s is not enabled", ifname,
```

Comment:
Suggest to change it to.. 

zlog_debug("ifname: %s for vrf: %s(id: %u) is not enabled", ifname,
                     vrf ? vrf->name : "NO VRF NAME", vrf_id);